canvas_data_warehouse/services.py

```diff
@@ -0,0 +1,69 @@
+'''Provide services of data graph.'''
+
+
+class CanvasDataService:
+    '''provide services for getting canvas data'''
+
+    STAFF_STATISTICS = 0
+    TRAINEE_STATISTICS = 1
+    FULL_TIME_TEACHER_TRAINED_COVERAGE = 2
+    TRAINING_HOURS_WORKLOAD_STATISTICS = 3
+
+    BY_DEPARTMENT = 0
+    BY_STAFF_TYPE = 1
+    BY_STAFF_TITLE = 2
+    BY_HIGHEST_DEGREE = 3
+    BY_AGE_DISTRIBUTION = 4
+
+    BY_DEPARTMENT = 0
+    BY_STAFF_TITLE = 1
+    BY_AGE_DISTRIBUTION = 2
+
+    BY_TOTAL_STAFF_NUM = 0
+    BY_TOTAL_TRAINING_HOURS = 1
+    BY_PER_CAPITA_TRAINING_HOURS = 2
+    BY_TOTAL_WORKLOAD = 3
+    BY_PER_CAPITA_WORKLOAD = 4
+
+    staff_grouping_type = [
+        {'type': BY_DEPARTMENT, 'name': '按学院'},
```

Comment:
If you need `statistics_type` in JSON format for the response, consider constructing it in the service method, construct it here will restraint its flexibility for further use.